src/Styleguide/Elements/Spacer.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react"
+import { Box } from "Styleguide/Elements/Box"
+import { SpaceProps } from "styled-system"
+
+export const Spacer: React.SFC<SpaceProps> = props => {
```

Comment:
@damassi - I'm wondering if it makes sense to combine this with `Separator` that I've added at some point. We can make it take a property weather to draw a line or not. What do you think?

Reply:
I think we should keep them separate; both of these items qualify as primitives.